src/components/Link.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {useHref, useLinkClickHandler} from 'react-router-dom';
+
+import Anchor from 'components/Anchor';
+
+/**
+ * Custom Link component using the design system component, replacing react-router's Link.
+ */
+const Link = React.forwardRef(({onClick, replace = false, state, target, to, ...rest}, ref) => {
+  const href = useHref(to);
+  const handleClick = useLinkClickHandler(to, {
+    replace,
+    state,
+    target,
+  });
+  return (
+    <Anchor
+      {...rest}
+      href={href}
+      onClick={event => {
+        onClick?.(event);
```

Comment:
I didn't know this syntax :exploding_head:

Reply:
me neither - copy pasted it straight from the react-router documentation!